lang/LangPrimSource/PyrArrayPrimitives.cpp

```diff
@@ -30,6 +30,9 @@ Primitives for Arrays.
 #include "SC_Constants.h"
 #include <string.h>
 
+#include "SC_Levenshstein.h"
+#include <algorithm>
```

Comment:
Oops! Sorry. I added it to the end and just forgot to move it.

(I didn't add string.h, but can change it if needed? I didn't want to do drive-by stuff)

Reply:
my rule is, if you're already moving it and it's an unequivocal improvement, just do it. but if you don't know it can be improved in the first place, or don't know whether the improvement is unequivocally better, it's better to leave it.